common.yml

```diff
@@ -439,10 +472,14 @@ functions:
         if [ "${INT_TEST}" = "true" ]; then
           export TOOLS_TESTING_INTEGRATION="true"
         fi;
+        MONGO_ARGS="${mongo_args}"
+        if [ "${USE_TLS}" = "true" ]; then
```

Comment:
`USE_TLS` is only set for the 4.4 tests.

Reply:
Not quite! The mongo tools aren't configured to run with TLS (only SSL), and so we don't need to set those TLS environment variables here.

Reply:
I mean we should set 
```
          export TOOLS_TESTING_SSL="true"
          export TOOLS_TESTING_INTEGRATION="true"
```
when ssl/tls is enabled. But currently this only checks USE_SSL flag but not USE_TLS

Reply:
I mean we should set 
```
          export TOOLS_TESTING_SSL="true"
          export TOOLS_TESTING_INTEGRATION="true"
```
when ssl/tls is enabled. But currently this only checks `USE_SSL` flag but not `USE_TLS`

Reply:
I'm passing in `USE_SSL` to `run tool integration tests`, so I don't think we need to do that. I think `USE_TLS` is carried over from an earlier function, which is why its values is `true` here, but I can add in your suggestion for improved readability.

Reply:
ah, cool! Yea, seems currently these two are not very clearly identified. It would be better to add it.

Reply:
ah, cool! Yea, seems currently these two are not very clearly identified. It would be better to add it.